t/04-decode-string.t

```diff
@@ -6,47 +6,52 @@ plan 3;
 
 my $encoded = q[_p~iF~ps|U_ulLnnqC_mqNvxq`@];
 
-is decode-polyline( $encoded ), [ { :lat(38.5), :lon(-120.2) }, { :lat(40.7), :lon(-120.95) }, { :lat(43.252), :lon(-126.453) } ], "String decoded OK";
+is-deeply decode-polyline( $encoded ), [
+    { lat => 38.5,     lon => -120.2   },
+    { lat => 40.7,     lon => -120.95  },
+    { lat => 43.252,   lon => -126.453 },
+], "String decoded OK";
 
 $encoded = q<yikzH{o`@zsHdP|nIsnBxiE{wFboBuxJwi@m|Sc_DonMmdGqcLqnLqcLycFpGonD~eBonCngJ|N`lWfyB~y]vnA|wFpnD~lE>;
 
-is decode-polyline( $encoded ), [
-    { 'lat'=>'51.67277', 'lon'=>'0.17166', },
-    { 'lat'=>'51.62335', 'lon'=>'0.16891', },
-    { 'lat'=>'51.5696',  'lon'=>'0.18677', },
-    { 'lat'=>'51.53715', 'lon'=>'0.22659', },
-    { 'lat'=>'51.51921', 'lon'=>'0.28702', },
-    { 'lat'=>'51.52605', 'lon'=>'0.39413', },
-    { 'lat'=>'51.55167', 'lon'=>'0.46829', },
-    { 'lat'=>'51.5935',  'lon'=>'0.53558', },
-    { 'lat'=>'51.66255', 'lon'=>'0.60287', },
-    { 'lat'=>'51.69916', 'lon'=>'0.6015',  },
-    { 'lat'=>'51.72724', 'lon'=>'0.58502', },
-    { 'lat'=>'51.7502',  'lon'=>'0.52734', },
-    { 'lat'=>'51.74765', 'lon'=>'0.40237', },
-    { 'lat'=>'51.72809', 'lon'=>'0.24445', },
-    { 'lat'=>'51.71533', 'lon'=>'0.20462', },
-    { 'lat'=>'51.68724', 'lon'=>'0.17166', },
+exit 1;
```

Comment:
Hi Alex. Firstly thanks muchly for taking a look at this I'm sorry it's taken me so long to reply I'd forgotten to keep an eye on my github messages.
I'm going to see if I can get emails sent. 
This `exit 1` was it is left in by accident because I think it's causing the test to explode. 

Other than that it's looking awesome :)